js-envs/src/main/scala/org/scalajs/jsenv/ExternalJSEnv.scala

```diff
@@ -25,9 +25,22 @@ abstract class ExternalJSEnv(
   /** Custom initialization scripts. */
   protected def customInitFiles(): Seq[VirtualJSFile] = Nil
 
-  protected class AbstractExtRunner(protected val classpath: CompleteClasspath,
-      protected val code: VirtualJSFile, protected val logger: Logger,
-      protected val console: JSConsole) {
+  protected class AbstractExtRunner(
+      protected val libs: Seq[ResolvedJSDependency],
+      protected val sjsCode: VirtualJSFile,
+      protected val code: VirtualJSFile) {
```

Comment:
I'm actually wondering if we shouldn't do it the other way around and have `sjsCode` just be a part of `libs`...